NEXTCLOUD.cmake

```diff
@@ -15,6 +15,28 @@ set( APPLICATION_VIRTUALFILE_SUFFIX "nextcloud" CACHE STRING "Virtual file suffi
 set( LINUX_PACKAGE_SHORTNAME "nextcloud" )
 set( LINUX_APPLICATION_ID "${APPLICATION_REV_DOMAIN}.${LINUX_PACKAGE_SHORTNAME}")
 
+option(DEV_BUILD "If enabled, name variables will be appended with safe variations of DEV_SUFFIX in order to reduce conflicts." OFF)
+
+if (NOT DEV_SUFFIX)
+    set(DEV_SUFFIX "Dev"
+        CACHE STRING
+        "Choose a suffix with which to append build name variables when DEV_BUILD is enable in order to prevent conflicts.
+        Characters other than a-z, A-Z, 0-9, \"_\", \".\", \"+\" and \"-\" will be stripped out where necessary."
+        FORCE)
+endif ()
+
+if (DEV_BUILD)
+    message(STATUS "DEV_BUILD is set to ${DEV_BUILD}, and name variables will be appended with safe variations of DEV_SUFFIX value \"${DEV_SUFFIX}\" to prevent conflicts.")
+    string(REGEX REPLACE [^a-zA-Z0-9\._+-] "" SUFFIX_STRIPPED "${DEV_SUFFIX}")
```

Comment:
Please drop that. We don't need that. The only suffixes that we need are Dev and Daily.

Reply:
No. Your use cases are not necessarily other people’s use cases. (For example, someone might want to do multiple comparison builds at the same time, or someone might use the Git branch name as a suffix.)